config/v1/types_apiserver.go

```diff
@@ -0,0 +1,61 @@
+package v1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// APIServer holds cluster-wide information about api-servers.  The canonical name is `cluster`
+type APIServer struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              APIServerSpec   `json:"spec"`
+	Status            APIServerStatus `json:"status"`
+}
+
+type APIServerSpec struct {
+	// servingCert is the TLS cert info for serving secure traffic.
+	// +optional
+	ServingCerts APIServerServingCerts `json:"servingCerts,omitempty"`
+	// clientCA is the certificate bundle for all the signers that you'll recognize for incoming client certificates
```

Comment:
that you'll recognize for incoming client certifiates in addition too cluster maintained signers